README.md

```diff
@@ -44,7 +44,7 @@ in order to avoid installing cuTile globally:
 
 ```
 python3 -m venv env
-source env/bin/activate
+source venv/bin/activate
```

Comment:
This change doesn't look right, I think the original is correct. The first command `python3 -m venv env` invokes the `venv` module to create an environment in a directory named `env`, so the path should start with `env/`